recipes/pylibjpeg-rle/recipe.yaml

```
@@ -0,0 +1,51 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+context:
+  name: pylibjpeg-rle
+  python_name: pylibjpeg_rle
+  version: 2.1.0
+  python_min: "3.10"
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/${{ python_name }}-${{ version }}.tar.gz
+  sha256: 3ccba7b912f5bc0facc0f2738a5a55dd843653d2f3c02cc31a09fb5efad4b9c7
+
+build:
+  number: 0
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  build:
+    - cargo-bundle-licenses
+    - ${{ compiler('rust') }}
+    - ${{ stdlib("c") }}
+  host:
+    - python ${{ python_min }}.*
+    - maturin >=1.0,<2.0
+    - pip
+  run:
+    - python
+    - numpy >=2.0
+
+tests:
+  - python:
+      imports:
+        - rle
+      pip_check: true
+      python_version: ${{ python_min }}.*
+
+about:
+  homepage: https://github.com/pydicom/pylibjpeg-rle
+  summary: Python bindings for a fast RLE decoder/encoder, with a focus on use as a plugin for pylibjpeg
+  license: MIT
+  license_file: LICENSE
```

```suggestion
  license_file: 
    - LICENSE
    - THIRDPARTY.yaml
```
We also have add the third-party licenses file here.